pallets/async-backing/src/lib.rs

```diff
@@ -94,6 +94,9 @@ pub mod pallet {
 
 		/// A way to get the current parachain slot and verify it's validity against the relay slot.
 		type GetAndVerifySlot: GetAndVerifySlot;
+
+		#[pallet::constant]
+		type ExpectedBlockTime: Get<Self::Moment>;
```

Comment:
Maybe add a comment that this is used by tools like chopsticks, to prevent future developers from accidentally removing it

Reply:
done